mysql-test/main/partition_error.test

```diff
@@ -16,7 +16,16 @@ let $MYSQLD_DATADIR= `SELECT @@datadir`;
 --echo #
 CREATE TABLE t1 (a int);
 CREATE OR REPLACE VIEW v1 AS SELECT * FROM t1;
---error ER_CHECK_NO_SUCH_TABLE
+--error ER_WRONG_OBJECT
+ALTER TABLE t1 EXCHANGE PARTITION p0 WITH TABLE v1;
+DROP VIEW v1;
+DROP TABLE t1;
+
```

Comment:
Could you add the following before the test you've added?
```
--echo #
--echo # MDEV-28599 EXCHANGE PARTITION on view causes ER_CHECK_NO_SUCH_TABLE instead of ER_WRONG_OBJECT
--echo #
```